### PR TITLE
LOG-3973: [elasticsearch-operator] Add config for branch release-5.7

### PR DESCRIPTION
**Pull request by periklis**
cc @jcantrill

### CODE REVIEW

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml [hunk 1]
```diff
@@ -93,13 +93,13 @@ operator:
...
 promotion:
-  name: "5.7"
+  name: "5.8"
   namespace: logging
 releases:
   latest:
     release:
       channel: fast
-      version: "4.12"
+      version: "4.13"
```
Comment (xperimental) [inline ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml:102]: If I'm interpreting the errors in the rehearsals correctly, then there's no "fast-4.13" yet. "candidate-4.13" seems to work, if I try out the API myself, so maybe we need to change the channel until 4.13 is released.

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**